src/mixins/date-mixin.ts

```diff
@@ -17,15 +17,15 @@ export default class DateMixin extends Vue {
    * @example "2021, 0, 1, 0, 0" -> "2021-01-01T08:00:00.000Z"
    * @example "2021, 6, 1, 0, 0" -> "2021-07-01T07:00:00.000Z"
    */
-  createUtcDate (year: number, month: number, day: number, hours: number = 0, minutes: number = 0): Date {
-    // in this local mixin, use `new Date()`
-    // but in app mixin, get date from server
-    const currentJsDate = new Date()
-    const date = new Date(currentJsDate.toLocaleString('en-US', { timeZone: 'America/Vancouver' }))
-
-    // update all date and time fields
-    date.setFullYear(year, month, day)
-    date.setHours(hours, minutes, 0, 0) // zero out seconds and milliseconds
+  createUtcDate (year: number, month: number, day: number, hours: number = 0, minutes: number = 0): Date {    // 1. create the new date in UTC
```

Comment:
It will look really nice if you can move comment 1 to below the function.

Reply:
Done!